src/EnergyPlus/DataSizing.hh

```diff
@@ -120,6 +120,7 @@ namespace DataSizing {
     // parameters for sizing
     constexpr int NonCoincident(1);
     constexpr int Coincident(2);
+    constexpr int Combination(3);
```

Comment:
If 1 airloop sizes coincident and others size noncoincident then reporting needs an alternate notation.